test/unit/org/apache/cassandra/index/sai/cql/NativeIndexDDLTest.java

```diff
@@ -264,16 +265,16 @@ public void shouldNotFailCreateWithTupleType()
     }
 
     @Test
-    public void shouldFailCreateWithInvalidCharactersInColumnName()
+    public void reproFailOnLongIndexName()
```

Comment:
Maybe this could be named `shouldFailOnLongIndexName`?

Reply:
It will be `shouldFail` when the long names will be prevented by purpose. Here it's a bug. I can add a comment.

Reply:
Makes sense, +1.